app/src/components/common/Table.vue

```diff
@@ -44,21 +61,90 @@
 </template>
 
 <script lang="ts">
-import { TableColumn } from '@/types'
+import { TableColumn, SortOrder, SortRule } from '@/types'
 import { Vue, Component, Prop } from 'vue-property-decorator'
 
-@Component
+@Component({
+  computed: {
+    SortOrder () {
+      return SortOrder
+    }
+  }
+})
 export default class Table extends Vue {
   @Prop({ required: true }) columns!: TableColumn[]
   @Prop({ required: true }) data!: Array<{ [key: string]: string }>
+  sortedData: Array<any> = []
```

Comment:
do you mostly sort an array of `User` isn't it?

Reply:
Changed type to { [key: string]: string } here in the recent commit.